src/operator/elemwise_op_common.h

```diff
@@ -205,16 +261,22 @@ struct ElemwiseGradUseNone {
   }
 };
 
-struct CloneGradient {
```

Comment:
Must have got nuked in a merge. I will replace.  Haibin, where is the unit test which would catch this?